src/main/java/com/aws/greengrass/componentmanager/builtins/ArtifactDownloader.java

```diff
@@ -131,12 +135,14 @@ public File download() throws PackageDownloadException, IOException, Interrupted
                     offset.set(0);
                     messageDigest.reset();
                     throw new ArtifactChecksumMismatchException(
-                            "Integrity check for downloaded artifact failed. " + "Probably due to file corruption.");
+                            "Integrity check for downloaded artifact failed. Probably due to the file changing"
+                                    + " after creating the component version",
```

Comment:
can you change this message? "Probably due to the file changing after creating the component version" is grammatically incorrect.

something like "Failed integrity check for the downloaded artifact. Artifact contents may have changed after component version was created"

Reply:
will update